app/globals.css

```
@@ -2,6 +2,9 @@
 @import "tw-animate-css";
 @plugin '@tailwindcss/typography';
 
+@import "@fuma-comment/react/preset.css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)